package.json

```diff
@@ -96,9 +97,9 @@
         "views": {
             "azure": [
                 {
-                    "id": "cosmosDBExplorer",
-                    "name": "Cosmos DB",
-                    "when": "config.cosmosDB.showExplorer == true"
+                    "id": "azureDatabasesExplorer",
+                    "name": "Azure Databases",
```

Comment:
Remove "Azure" from this name. It's already under the "Azure" viewsContainer and would look weird if they only have this extension installed since VS Code appends the viewContainer name to the view name:
![Screen Shot 2020-04-22 at 4 06 39 PM](https://user-images.githubusercontent.com/11282622/80042250-606d2700-84b3-11ea-8c7d-7e9453b3ea3e.png)